crypto/ocsp/ocsp_test.cc

```diff
@@ -1385,6 +1385,16 @@ TEST_P(OCSPNonceTest, OCSPNonce) {
   }
   EXPECT_EQ(OCSP_check_nonce(ocspRequest.get(), basicResponse.get()),
             t.nonce_check_status);
+
+  // Check that nonce copying from |req| to |bs| also works as expected.
+  if (t.nonce_check_status == OCSP_NONCE_RESPONSE_ONLY ||
+      t.nonce_check_status == OCSP_NONCE_BOTH_ABSENT) {
+    EXPECT_EQ(OCSP_copy_nonce(basicResponse.get(), ocspRequest.get()), 2);
```

Comment:
what do we expect `OCSP_check_nonce` to be after this copying no-op?

Reply:
The check's behavior will be identical to L1386 since there's no nonce to be copied. It seemed a bit redundant, but I can add the extra check here for parity across the if else statements.